docs/installation-deployments.md

```diff
@@ -39,7 +39,20 @@ node build/start-server.js
 ```
 
 # Deploy using Docker
-Docs to be written, please help!
+Run using Docker directly:
+```
+sudo docker run -d -p 8787:8787 portkeyai/gateway
+```
+
+# Deploy using Docker Compose
+1. Download compose file from the repo:
+```
+wget "https://raw.githubusercontent.com/Portkey-AI/gateway/main/docker-compose.yaml"
+```
+2. Run:
```

Comment:
Nitpicking very small change: Can we also mention the default container port that will be exposed? Default is 8787